src/avfoundation.cs

```diff
@@ -13552,6 +13849,35 @@ interface AVContentKeyResponse {
 		AVContentKeyResponse _InitWithAuthorizationToken (NSData authorizationTokenData);
 	}
 
+	[TV (14,5), Mac (11,3), iOS (14,5), Watch (7,4)]
+	[BaseType (typeof(NSObject))]
+	interface AVContentKeySpecifier
+	{
+		[Static]
+		[Export ("contentKeySpecifierForKeySystem:identifier:options:")]
+		AVContentKeySpecifier GetContentKeySpecifier (AVContentKeySystem keySystem, NSObject contentKeyIdentifier, NSDictionary<NSString, NSObject> options);
+
+		[Export ("initForKeySystem:identifier:options:")]
+		IntPtr Constructor (AVContentKeySystem keySystem, NSObject contentKeyIdentifier, NSDictionary<NSString, NSObject> options);
```

Comment:
Same, can we have a strong dict?

Reply:
https://github.com/xamarin/xamarin-macios/issues/10904